src/neofs_testlib/hosting/docker_host.py

```diff
@@ -137,8 +137,8 @@ def delete_storage_node_data(self, service_name: str, cache_only: bool = False)
         volume_path = volume_info["Mountpoint"]
 
         shell = self.get_shell()
-        meta_clean_cmd = f"rm -rf {volume_path}/meta*/*"
-        data_clean_cmd = f"; rm -rf {volume_path}/data*/*" if not cache_only else ""
+        meta_clean_cmd = f"sudo rm -rf {volume_path}/meta*/*"
+        data_clean_cmd = f"; sudo rm -rf {volume_path}/data*/*" if not cache_only else ""
```

Comment:
Dangerous. It's OK for GH runners, but what if you're running this locally and there is a bug in `{volume_path}`?

Reply:
Are there any other safe way to clean storage node data? I can add a validation that this code is running on a GH runner, and if not - fail with an assertion error. Or I can hardcode here a prefix path to volumes, but in this way - it still can potentially fail on a non GH runner.

Reply:
i've hardcoded the path here

Reply:
You can change ownership of this folder/files and then delete from a regular user.

Reply:
You mean during a deploy phase on a GH runner? Or right here? In both ways, what's the difference? If volume_path is corrupted we will remove smth we don't want to remove. I like the hardcoded option a little bit more, since we explicitly specify what we want to delete.

Reply:
Likely during the env setup. The difference is that then you don't need `sudo` for destructive actions, you'll just `chown` and that's it, `rm` would work with user privileges.

Reply:
That's correct, but I can chown wrong files/directories and then successfully remove them with user privileges. The same way as with sudo but with two commands instead of one.

Reply:
In any case, I'm not sure it is a good idea to move this code (either chown/rm or sudo rm) to the env setup, since it is closely related this particular command and can be considered as a part of the test

Reply:
The best thing would probably be doing `docker exec`, that'd be contained. But we no longer have a proper shell inside of this container. But you can attach a volume to another container like

`docker run --volumes-from s01 alpine ls /storage`

Or `rm -fr`, that'd be exactly what you need, you have complete root privileges there and you're not messing with host's FS.

Reply:
I took a closer look at this volume contents, and there are these entries:

```
blobovnicza0
blobovnicza1
fstree0
fstree1
meta0
meta1
pilorama0
pilorama1
```

The function ```delete_storage_node_data```clears only **meta** and **data** files. But there are no **data** files. Are this code a little bit outdated? Should we remove all above entries here?

Reply:
bbcz/fstree, that's where the data is stored now. But if you're to delete those then pilorama state doesn't make much sense either (although it's a separate thing). The function also has `cache_only` mode when it should only drop metabase.

Reply:
Updated the code to use docker client to remove meta and data files, pls take a look again